etc/format_code.jl

```diff
@@ -7,15 +7,41 @@ Pkg.add(name="JuliaFormatter", version="1")
 
 using JuliaFormatter
 
+
+# ------------------------------------------------------------------------------
 # Disclaimer:
-# The goal should be to work with 
-# https://github.com/julia-actions/julia-format
-# . However, currently too many files have broken formatting and since there
-# are many ongoing pull requests, we will need to extend proper formatting to
-# the entire codebase in a step by step fashion.
 #
-# In case you format some code, also add the commit hash to
-# .git-blame-ignore-revs for `git blame` to ignore these commits.
+# To ensure consistent formatting, we employ the package `JuliaFormatter.jl`. The
+# `.JuliaFormatter.toml` file in the repository defines the desired formatting style.
+#
+# To format your files, first add the `JuliaFormatter.jl` package in Julia. You
+# can then format individual files via:
+#
+# using JuliaFormatter
+# format_file("path/to/file/file.jl")
+#
+# To format an entire folder (recursively), use `format("path/to/folder")`. Note
+# that both `format` and `format_file` return a `Bool`; you may need to repeat the
+# command until it returns `true`. This is a known shortcoming of the current
+# `JuliaFormatter` package.
+#
+# Ultimately, we aim to use the workflow provided by https://github.com/julia-actions/julia-format.
+# Instead of performing a single large reformatting of the entire OSCAR code base, we are extending
+# coverage incrementally. A custom CI job ensures that already-compliant parts of OSCAR
+# remain consistent with the defined style. Whenever the scope of this CI is extended
+# - for example, to verify that code in a new experimental module meets our formatting
+# requirements - special care must be taken. Details on how to extend the scope of this
+# CI mechanism are described below. All other formatting changes can be handled like regular commits.
+#
+# Extending the JuliaFormatter CI
+# 1) Create a commit that contains only the formatting changes automatically
+#    generated by JuliaFormatter - no other modifications.
+# 2) Create a second commit with the following updates:
+#    a) Add the commit hash from step 1 to `.git-blame-ignore-revs`, so that
+#       `git blame` ignores these purely stylistic edits.
```

Comment:
In practice this second commit either needs to be in a separate PR; or we need to make sure the PR with both commits is merged via a "true" merge -- not a rebase or squash, as both of those change the commit hashes.

Perhaps we should explain that? Or just describe that it should be done in two PRs, and why.